lib/auth/bot_test.go

```diff
@@ -32,7 +35,11 @@ import (
 	"golang.org/x/crypto/ssh"
 )
 
-func TestBotCreateFeatureDisabled(t *testing.T) {
+// TestServer_createBot_FeatureDisabled ensures that you cannot create a bot
+// when the appropriate license does not exist. It is a separate test from
+// TestServer_createBot as `modules.SetTestModules` does not work with parallel
+// tests.
+func TestServer_createBot_FeatureDisabled(t *testing.T) {
```

Comment:
Can we nix the `_` characters? They just feel so out of place in Go code.

Reply:
Sure, I'll go ahead and do that here. I do think that we ought to have a conversation as a team at some point about test naming, some conformity will help us improve test discoverability a fair bit. I renamed these tests because I had struggled to find them.

Reply:
[3298bff](https://github.com/gravitational/teleport/pull/15117/commits/3298bff0b97275e7e7af2858d57597abdc446a20)